src/components/ToolTip.tsx

```diff
@@ -93,8 +97,83 @@ export default function ({
   const { xs } = useBreakpoints()
   useClickOutside(childrenRef, () => setIsShow(false))
 
+  const [node, setNode] = useState<VNode | null>(null)
+
+  const positionTooltip = useCallback(
+    (event: MouseEvent) => {
+      if (position !== 'floating') return
+
+      const x = event.pageX
+      const y = event.pageY
+      const element = document.getElementById('root')
+      const parent = childrenRef.current
+      const positionX = (xs ? x * 0.5 : x * 0.95) + 'px;'
+      const positionY = y + 0.5 + 'px;'
+      const portalWrapper = width('lg:w-card', 'w-6/12')
+
+      const portalStyles =
+        `position:absolute; z-index: 49;` +
+        'left:' +
+        positionX +
+        'top:' +
+        positionY
+
+      if (!parent) return
+      const parentXStart = parent.getBoundingClientRect().left + window.scrollX
+      const parentXEnd = parentXStart + parent.offsetWidth
+      const parentYStart = parent.getBoundingClientRect().top + window.scrollY
+      const parentYEnd = parentYStart + parent.offsetHeight
+      if (!element) return
+
+      if (
+        x < parentXStart ||
+        x > parentXEnd ||
+        y < parentYStart ||
+        y > parentYEnd
+      ) {
+        setIsShow(false)
+        setNode(null)
+      }
+
+      const node = createPortal(
+        <div style={portalStyles} className={portalWrapper}>
+          <div className={tooltipWrapper}>
+            <div
+              className={tooltipClasses(xs, isShow, 'bottom')}
+              style={{ visibility: isShow ? 'visible' : 'collapse' }}
+            >
+              <HighlightedText>{text}</HighlightedText>
+              {arrow && <div className={triangle('bottom')} />}
+            </div>
+          </div>
+        </div>,
+        element
+      )
+      setIsShow(true)
+      setNode(node)
+    },
+    [arrow, isShow, xs, position, text]
+  )
+
+  const isFloating = position === 'floating'
+
+  useEffect(() => {
+    if (!isFloating) return
+    const element = childrenRef.current
+    if (!element) return
+
+    element.addEventListener('mousemove', positionTooltip)
+    element.addEventListener('mouseleave', positionTooltip)
```

Comment:
Have you tried `mouseover`? https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseover_event

Reply:
Yep, thx